src/item/presentation/item.read.adapter.ts

```
@@ -29,4 +29,11 @@ export class ItemReadAdapter {
 
         return new ResponseItemForm(articleList);
     }
+
+    @Post('validate')
+    async validateItems(@Body() body: { organId: number; ids: number[] }): Promise<Boolean> {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)